src/borg/archiver.py

```diff
@@ -393,9 +397,16 @@ def create_inner(archive, cache):
             create_inner(None, None)
         return self.exit_code
 
-    def _process(self, archive, cache, matcher, exclude_caches, exclude_if_present,
-                 keep_exclude_tags, skip_inodes, path, restrict_dev,
-                 read_special=False, dry_run=False, st=None):
+    def _process(self, *, path, **kwargs):
+        try:
+            self._process_real(path=path, **kwargs)
```

Comment:
that doubles recursion depth. not sure how relevant for practical purposes, but we already have a ticket about that.